xmtp_mls/src/utils/id.rs

```diff
@@ -8,13 +10,13 @@ pub fn serialize_group_id(group_id: &[u8]) -> String {
 pub fn calculate_message_id(
     group_id: &[u8],
     decrypted_message_bytes: &[u8],
-    sender_account_address: &str,
+    inbox_id: InboxId,
```

Comment:
In another PR I actually just remove the sender identity from the calculation altogether. I don't think we need it for uniqueness, since we have the `idempotency_key`